src/web/components/SubNav/Inner.tsx

```diff
@@ -154,7 +154,7 @@ export const Inner: React.FC<{
                     [selected]: link.title === currentNavLink,
                 })}
                 href={link.url}
-                data-link-name={`nav2 : subnav : ${link.title}`}
+                data-link-name={`nav2 : subnav : ${link.url.slice(1)}`} // Remove starting slash
```

Comment:
Is there value in something like:
```
function trimLeadingSlash(url) { if (url.substr(0, 1) === '/') return url.slice(1); }
```
?

It's a waste if we're sure the first character will always be a slash but might be a nice guard if not.

Reply:
It's useful even if only from a comprehension POV